FirebaseAuth.podspec

```diff
@@ -0,0 +1,54 @@
+Pod::Spec.new do |s|
+  s.name             = 'FirebaseAuth'
+  s.version          = '4.3.2'
+  s.summary          = 'The official iOS client for Firebase Authentication'
+
+  s.description      = <<-DESC
+Firebase Authentication allows you to manage your own account system without any backend code. It
+supports email and password accounts, as well as several 3rd party authentication mechanisms.
+                       DESC
+
+  s.homepage         = 'https://firebase.google.com'
+  s.license          = { :type => 'Apache', :file => 'LICENSE' }
+  s.authors          = 'Google, Inc.'
+
+  s.source           = { :git => 'https://github.com/firebase/firebase-ios-sdk.git', :tag => s.version.to_s }
+  s.social_media_url = 'https://twitter.com/Firebase'
+  s.ios.deployment_target = '7.0'
+  s.osx.deployment_target = '10.10'
+
+  s.cocoapods_version = '>= 1.4.0.beta.2'
+  s.static_framework = true
+  s.prefix_header_file = false
+
+  source = 'Firebase/Auth/Source/'
+  s.source_files = source + '**/*.[mh]'
+  s.osx.exclude_files =
```

Comment:
Please use square brackets for lists that extend over multiple lines.

Reply:
done